### PR TITLE
Adapt Proximity Sensor IDL tests to latest spec

### PR DESCRIPTION
**Pull request by xiuqijix**
and move IDL to a standalone file

### CODE REVIEW

**Comment by w3c-bots** [comment]
# Build PENDING

Started: None
Finished: None





View more information about this build on:

- [WPT PR Status](http://pulls.web-platform-tests.org/build/22770)
- [TravisCI](https://travis-ci.org/w3c/web-platform-tests/builds/328929547)

File: proximity/idlharness.https.html [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+"use strict";
+
+function doTest([dom, generic_sensor, proximity]) {
+  const idl_array = new IdlArray();
+  idl_array.add_untested_idls(dom);
+  idl_array.add_untested_idls('interface EventHandler {};');
+  idl_array.add_idls(generic_sensor, { only: ['Sensor'] });
+  idl_array.add_idls(proximity);
+  idl_array.add_objects({
+    Proximity: ['new Proximity();']
```
Comment (Honry) [inline proximity/idlharness.https.html]: 's/Proximity/ProximitySensor'

Comment (kaixinjxq) [reply proximity/idlharness.https.html]: Thanks @Honry, changed.

**Review by Honry — APPROVED** [review]
Thanks, LGTM

**Pull request merged**